src/rewriting.jl

```diff
@@ -20,26 +21,39 @@ RewritingSystem written as a list of pairs of `Word`s together with the ordering
 struct RewritingSystem{W<:AbstractWord, O<:Ordering} <: AbstractRewritingSystem{W, O}
     rwrules::Vector{Pair{W,W}}
     order::O
+    act::BitArray{1}
 end
 
+RewritingSystem(rwrules::Vector{Pair{W,W}}, order::O) where{W<:AbstractWord, O<:Ordering} = RewritingSystem(rwrules, order, trues(length(rwrules)))
+
 rules(s::RewritingSystem) = s.rwrules
+active(s::RewritingSystem) = s.act
+arules(s::RewritingSystem) = rules(s)[active(s)]
```

Comment:
this allocates a copy. If we just want to iterate, then a generator would do: `(r for (i,r) in enumerate(rules(s)) if isactive(s, i))`